acceptance-tests/lib/cmd.ts

```diff
@@ -37,53 +36,46 @@ function createProcess(cliPath, cliVersion, args = [], env = null) {
   // This works for node based CLIs, but can easily be adjusted to
   // any other process installed in the system
   return spawn(processCommand, args, {
-    env: Object.assign(
-      {
-        NODE_ENV: 'test',
-        preventAutoStart: false,
-        PATH, // This is needed in order to get all the binaries in your current terminal
-      },
-      env
-    ),
+    env: {
+      NODE_ENV: 'test',
+      preventAutoStart: false,
+      PATH, // This is needed in order to get all the binaries in your current terminal
+      ...env,
+    },
     stdio: [null, null, null, 'ipc'], // This enables interprocess communication (IPC)
   });
 }
 
-/**
- * Creates a command and executes inputs (user responses) to the stdin
- * Returns a promise that resolves when all inputs are sent
- * Rejects the promise if any error
- * @param {string} processPath Path of the process to execute
- * @param {Array} args Arguments to the command
- * @param {Array} inputs (Optional) Array of inputs (user responses)
- * @param {Object} opts (optional) Environment variables
- */
 function executeWithInput(
-  cliPath,
-  cliVersion,
-  args = [],
-  inputs = [],
-  opts = {}
+  config: TestConfig,
+  args: string[] = [],
+  inputs: string[] = [],
+  opts: any = {}
 ) {
   if (!Array.isArray(inputs)) {
     opts = inputs;
     inputs = [];
   }
 
-  if (global.config.headless) {
-    opts.env = { BROWSER: 'none' };
-  }
+  // Prevent the browser from opening when `open` is called
+  opts.env = { BROWSER: 'none' };
```

Comment:
I removed the check for the `headless` config value, because I can't think of why we would want to open browser windows.  We don't plan on interacting with them from the tests and all it does is make your system borderline unusable for the duration of the tests.